core/templates/services/keyboard-shortcut.service.spec.ts

```diff
@@ -28,14 +38,45 @@ describe('Keyboard Shortcuts', () => {
   var searchBar = document.createElement('input');
   var categoryBar = document.createElement('select');
 
+  var openQuickReferenceSpy;
+
   var mockWindow = {
     location: {
       href: ''
     }
   };
 
-  const windowRef = new WindowRef();
-  const keyboardShortcutService = new KeyboardShortcutService(windowRef);
+  var windowRef;
+  var keyboardShortcutService;
+  var component: KeyboardShortcutHelpModalComponent;
+  var fixture: ComponentFixture<KeyboardShortcutHelpModalComponent>;
+  var ngbActiveModal: NgbActiveModal;
+  var ngbModal: NgbModal;
```

Comment:
Can we convert all the `var` to `let` in this file?

Reply:
yes, done